src/transformers/modeling_utils.py

```diff
@@ -1666,6 +1666,51 @@ def get_memory_footprint(self, return_buffers=True):
             mem = mem + mem_bufs
         return mem
 
+    def to(self, *args):
+        r"""
+        Overrides the `.to` function from `nn.Module` to raise a ValueError if `8-bit` models are tried to be set on a
+        new device or `dtype`
+        """
```

Comment:
Let's remove this otherwise this will appear in the documentation of `PreTrainedModel`. The code is self explanatory.

Reply:
Thanks for the explanation! Removed all unneeded docstrings in 
7d6e6ff